src/app/pages/test-results-search/test-results-search.scss

```diff
@@ -61,7 +61,6 @@
     width: 810px;
     z-index: 100;
     padding: 16px 12px;
-    width: 100%;
```

Comment:
I would assume this one should remain and the other ```width: 810px;``` needs to go, I'd imagine the last width specified is the one that is used. But if tested and this is not the case then happy to approve